README.md

```diff
@@ -41,6 +57,27 @@ If you want to find a record using it's slug, a custom finder is provided by the
 $example = $this->Examples->find('slugged', ['slug' => $slug]);
 ```
 
+## Configuration
+
+Slug comes with the following configuration options:
+
+- ``field``: name of the field (column) to hold the slug. Defaults to ``slug``.
```

Comment:
Should be single "`" not double as per markdown format not reStructuredText format :)